src/br/com/harbitech/school/course/Course.java

```diff
@@ -0,0 +1,66 @@
+package br.com.harbitech.school.course;
+
+import br.com.harbitech.school.validation.ValidationUtil;
+import br.com.harbitech.school.subcategory.SubCategory;
+
+import java.time.OffsetDateTime;
+
+public class Course {
+
+    private Long id;
+    private String codeUrl;
+    private String name;
```

Comment:
```suggestion
    private String name;
    private String codeUrl;
```

TOC, inverter os campos para manter o padrão com categoria e subcategoria